.jenkins/pytorch/win-build.sh

```diff
@@ -118,6 +129,8 @@ set CMAKE_GENERATOR=Ninja
 
 if not "%USE_CUDA%"=="1" (
   if "%REBUILD%"=="" (
+    set CC=clang-cl
```

Comment:
I'm not sure whether we should change all windows builds to use clang-cl or set up a new build spec.